base/applications/cmdutils/taskkill/taskkill.c

```diff
@@ -315,6 +455,11 @@ static int terminate_processes(void)
     {
         WCHAR *p = task_list[i];
         BOOL is_numeric = TRUE;
+        DWORD *pkill_list = HeapAlloc(GetProcessHeap(), 0, pid_list_size * sizeof(DWORD));
```

Comment:
See my comment in the other function concerning `pkill_list` (it also applies there).